google-cloud-asset/samples/acceptance/quickstart_test.rb

```diff
@@ -1,4 +1,4 @@
-# Copyright 2020 Google, LLC
+# Copyright 2021 Google, LLC
```

Comment:
Oh, one other thing. Please don't update the copyright years. They should reflect when the file was created.

Reply:
done